### PR TITLE
Logos and college names, with styles!

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
- Fixed a typo in the FAQs
- Added a blank column to the Leaderboard table to house the college logos and moved college logo into the new column
- Added college name to column containing first name
- Added styling for college logo and college name
- Added actual college names and logos
- Updated API call to Rogue to filter by RRR run and id

Looks like this:
domino-logos

#### How should this be reviewed?
Is 40x40 a good size for the images? I did some styling stuff someone should probably make sure I didn't do anything crazy.

#### Relevant tickets
[Card](https://www.pivotaltracker.com/story/show/155900210)

#### Checklist
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h1>FAQ</h1>
               </header>
               <div class="card__content">
                 <h4>When does the leaderboard update?</h4>
-                <p>Every Monday afernoon! Which means if you want your numbers to be counted every week you need to submit your weekly update <a href="https://www.dosomething.org/us/campaigns/rinse-recycle-repeat-college-competition" target="_blank">here</a> by Sunday night.</p>
+                <p>Every Monday afternoon! Which means if you want your numbers to be counted every week you need to submit your weekly update <a href="https://www.dosomething.org/us/campaigns/rinse-recycle-repeat-college-competition" target="_blank">here</a> by Sunday night.</p>
```
Comment (chloealee) [inline site/index.html:54]: 🕵️‍♀️

**Review by chloealee — APPROVED** [review]
@katiecrane this looks AMAZING!!!!!!

File: site/index.html (hunk 1 above)
Comment (sbsmith86) [inline site/index.html:54]: 👀

**Review by sbsmith86 — APPROVED** [review]
looks dope.

**Pull request merged**